devtools/flake.nix

```diff
@@ -32,7 +32,7 @@
               vscodeWithExtensions = pkgs.vscode-with-extensions.override {
                 vscodeExtensions = with pkgs.vscode-extensions;
                   [
-                    matklad.rust-analyzer
+                    rust-lang.rust-analyzer
```

Comment:
matklad's rust-analyser is deprecated, nix gives us an error and forces replacement with the rust-lang version.